src/example_package/gui/app.py

```diff
@@ -0,0 +1,70 @@
+from kivy.app import App
+from kivy.properties import StringProperty
+from kivy.uix.boxlayout import BoxLayout
+
+
+class SumioApp(App):
+    kv_file = './kivy/app.kv'
+
+    def build(self):
+        return MainScreen()
+
+
+class MainScreen(BoxLayout):
+
+    def __init__(self, **kwargs):
+        super(MainScreen, self).__init__(**kwargs)
+        self.update_content('Quick Start')
+
+    def update_content(self, item):
+        content = self.ids.content
+        content.clear_widgets()
+        if item == 'Quick Start':
```

Comment:
This checking if item equals to these very specific strings really hurts my heart 😢 
Do you think it could be changed into a string-based enum? Of course, it would require all functions invoking `update_content` to know the enum value, But this information could be hidden in the invoking component.
This way we can change the text we show to the user without having to mend the code.

Reply:
Although to be honest I'm not sure if it's desirable / needed to make things like that at this stage of our project. What do you think?